tests/console/clamav.pm

```diff
@@ -7,7 +7,16 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Simple clamav test for SLE FIPS and openSUSE
+# Summary: check freshclam and clamscan against some fake virus samples
+# - install clamav and refresh the database using freshclam
+# - setup a swap partition on JeOS
+# - verify the databases (main, bytecode and daily) using sigtool
+# - override default systemd timeout (from 90s to 400s)
```

Comment:
not sure if this make sense in "coverage" info

Reply:
good point! I have modified/removed that part :)

Reply:
Thank you :)